src/components/html/EcVariations.html

```diff
@@ -0,0 +1,21 @@
+<div class="ec-product__variations" v-if="variations">

```

Comment:
aqui você deveria usar a classe igual ao nome do componente
```suggestion
<div class="ec-variations" v-if="variations">
```